src/main/java/ca/corefacility/bioinformatics/irida/repositories/analysis/submission/AnalysisSubmissionRepositoryImpl.java

```diff
@@ -95,7 +97,6 @@ public AnalysisSubmissionRepositoryImpl(DataSource dataSource, @Qualifier("outpu
 	 * {@inheritDoc}
 	 */
 	@Override
-	@SuppressWarnings("unchecked")
```

Comment:
Removed uncessary suppress warnings. Rest of the changes in the file are due to the formatter.